src/platform/plugins/shared/unified_doc_viewer/public/components/content_framework/section/section.tsx

```diff
@@ -62,39 +63,43 @@ export function ContentFrameworkSection({
   hasPadding = true,
 }: ContentFrameworkSectionProps) {
   const [accordionState, setAccordionState] = useState<EuiAccordionProps['forceState']>(forceState);
+  const renderActions = () => {
+    const size = 'xs';
+    return (
+      <>
```

Comment:
This is not necessary is it?

```suggestion

```

Reply:
There's probably a few of those around in the changed files, so could clean those up while we are at it.